src/modules/movie-cards.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable no-underscore-dangle */
```

Comment:
Great job team 🤝

- [ ] As requested by the previous reviewer, kindly solve this linter error instead of disabling it on this file. Linter checks help you as a developer to write code that follows best practices and it's therefore recommended that you don't disable them. Please rectify this.

Reply:
Hello, @1GORDON  I have tried to resolve this error but unfortunately when I remove the underscores the application crashes.